spec/services/weather_service_spec.rb

```diff
@@ -16,13 +16,10 @@
     it 'should return forcast data for a given latitude/longitude' do
       forecast_call = WeatherService.forecast_data(39.7456, -97.0892)
 
-      expect(forecast_call[:properties][:periods][0][:temperature]).to eq(58)
       expect(forecast_call[:properties][:periods][0][:temperature]).to be_an(Integer)
 
```

Comment:
I agree with removing these since the data was updating with current things. better to just test for structure!